package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
There will be no version 1.3.0 until release, something more like 1.3.0-SNAPSHOT I guess would do. At least that's what I used to in Java/Maven repositories. I am not sure how does *npm* handle this, though...